### PR TITLE
19-Extract-and-label-data-based-on-user-input

### PR DESCRIPTION
**Pull request by 1AhmedYasser**

### CODE REVIEW

File: GUI/src/pages/Training/IntentsFollowupTraining/RegexDetail.tsx [hunk 1]
```diff
@@ -233,15 +263,50 @@ const RegexDetail: FC = () => {
...
     input.type = 'file';
     input.accept = '.csv';
+    input.onchange = ((e) => {
+      const files = (e.target as HTMLInputElement).files ?? [];
+
+      if (!files) {
+        return;
+      }
+      const file: File = files[0];
+      console.log(file);
```
Comment (Minwasko) [inline GUI/src/pages/Training/IntentsFollowupTraining/RegexDetail.tsx:279]: Please remove this `console.log`

File: GUI/src/pages/Training/IntentsFollowupTraining/RegexDetail.tsx [hunk 2]
```diff
@@ -233,15 +263,50 @@ const RegexDetail: FC = () => {
...
+      if (file) {
+        const fileReader = new FileReader();
+        fileReader.onload = function (event) {
+            const csvOutput: string = event?.target?.result as string ?? '';
+            console.log(csvOutput)
+            let result = Papa.parse(csvOutput);
+            const data: string[] = result.data as string[] ?? []
+            console.log(data)
```
Comment (Minwasko) [inline GUI/src/pages/Training/IntentsFollowupTraining/RegexDetail.tsx:287]: Please remove this `console.log`

File: GUI/src/pages/Training/IntentsFollowupTraining/RegexDetail.tsx [hunk 3]
```diff
@@ -233,15 +263,50 @@ const RegexDetail: FC = () => {
...
+            data.forEach((e: string) => {
+              regex?.examples.push(e[0]);
+            })
+            const res = regex && regex.examples.map((e, index) => ({ id: index, value: e }));
+            console.log(res);
```
Comment (Minwasko) [inline GUI/src/pages/Training/IntentsFollowupTraining/RegexDetail.tsx:292]: Please remove this `console.log`

**Review by trevorling — APPROVED** [review]

**Review by Minwasko — APPROVED** [review]
Once console.logs are removed it looks ok 👍

**Pull request merged**